server.py

```diff
@@ -1176,6 +1177,13 @@ async def kills_cmd(ctx: ContextType):
         kills = [int(x) for x in f.read().split()]
     await ctx.reply(msg.format(kills, sum(kills)))
 
+@command("winstest")
+async def killstest_cmd(ctx: ContextType):
+    """Display the cumulative number of wins for the year-long challenge."""
+    msg = "A20 Heart kills in 2022: Total: {0} - Ironclad: {1} - Silent: {2} - Defect: {3} - Watcher: {4}"
```

Comment:
Neat little trick: you can do `"{0.all_character_count}, {0.ironclad_count}"` etc., and then pass `msg.format(run_stats.wins)`!